layouts/dpg-standard/single.html

```diff
@@ -0,0 +1,6 @@
+{{ define "main" }}
+
+{{ partial "alert.html" . }}
+{{ partial "default.html" . }}
+
+{{ end }}
```

Comment:
Do we need a [single page layout](https://gohugo.io/templates/single-page-templates/) for this template? Is this used when someone goes to view the DPG page itself, e.g.  `Home > dpg-indicators > 2`?

Reply:
In this scenario, the single-page layout is not needed. I removed it in commit 3cd9bec374b57798835d79c620de4c2ccae80d1e. We could add it later if we wanted each DPG Indicator page to have a unique, custom layout – but right now, we are using the default layout and not doing anything with it. We can continue to use the standard default layout for each Indicator page.

The `type: dpg-standard` should be used on the index page of the `/indicators/` category only. This gives us the desired effect:

![Screenshot of working DPG Indicators layout on the index page of the DPG Indicators section](https://user-images.githubusercontent.com/4721034/183155503-93848cbe-0240-4be7-9d25-668901b340ff.png "Screenshot of working DPG Indicators layout on the index page of the DPG Indicators section")